x-pack/legacy/plugins/fleet/index.ts

```diff
@@ -57,11 +57,13 @@ export function fleet(kibana: any) {
     config: () => config,
     configPrefix: CONFIG_PREFIX,
     init(server: any) {
-      server.newPlatform.setup.plugins.encryptedSavedObjects.registerType({
-        type: 'enrollment_api_keys',
-        attributesToEncrypt: new Set(['api_key']),
-        attributesToExcludeFromAAD: new Set(['enrollment_rules']),
-      });
+      // TODO https://github.com/elastic/kibana/issues/53199
+      // server.newPlatform.setup.plugins.encryptedSavedObjects.registerType({
```

Comment:
will commenting this out affect the current functionality of policy api keys?

Reply:
It's going to stop encrypting api keys, it's still going to work but it's not secure and production ready